target/linux/rockchip/image/sd.bootscript

```diff
@@ -0,0 +1,8 @@
+setenv bootargs "console=ttyS2,1500000 console=tty1 earlycon=uart8250,mmio32,0xff1a0000 root=PARTUUID=@ROOT@-02 rw rootwait"
```

Comment:
Any reason for hardcoding the partuuid instead of getting it from the boot medium via `part uuid` ?

Reply:
because this mechanism (more specificly ${mmc_bootdev} ) is sunxi-specific: https://lists.denx.de/pipermail/u-boot/2018-July/334837.html

I don't like it either, but I don't know about a better solution right now.

Reply:
I'm asking about this `root=PARTUUID=@ROOT@-02`, nothing about `$mmc_bootdev`.

Reply:
I need to specify the MMC device when calling part uuid. This needs mmc_bootdev again.
But yes, I can remove that hardcoding.